externs/browser/w3c_indexeddb.js

```diff
@@ -316,12 +379,13 @@ IDBRequest.prototype.errorCode;  // readonly
 /** @type {!DOMError} */
 IDBRequest.prototype.error; // readonly
 
-/** @type {Object} */
+/** @type {?Object} */
 IDBRequest.prototype.source; // readonly
 
-/** @type {IDBTransaction} */
+/** @type {!IDBTransaction} */
```

Comment:
Not backward compatible, but probably not a problem.
I'm willing to take it as long as none of our internal stuff breaks.